src/Containers/Microsoft.NET.Build.Containers/Resources/Strings.Designer.cs

```diff
@@ -105,6 +105,15 @@ internal class Strings {
             }
         }
         
+        /// <summary>
+        ///   Looks up a localized string similar to CONTAINER2020: {0} does not specify a registry and will be pulled from Docker Hub. Please prefix the name with the image registry, for example: '{1}/<image>'..
+        /// </summary>
+        internal static string BaseImageNameRegistryFallback {
```

Comment:
@baronfel I don't know how to generate this file, so I added the property manually to get the code to compile.

Reply:
Odd, the build should have generated the file. If this manual version was a problem then the CI would have flagged it, and it didn't, so I suppose your version was exactly correct :)